core/src/main/java/io/grpc/internal/PickFirstLoadBalancerProvider.java

```diff
@@ -33,15 +33,15 @@
  * down the address list and sticks to the first that works.
  */
 public final class PickFirstLoadBalancerProvider extends LoadBalancerProvider {
-  public static final String GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS =
-      "GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS";
+  public static final String GRPC_PF_USE_HAPPY_EYEBALLS = "GRPC_PF_USE_HAPPY_EYEBALLS";
```

Comment:
@ejona86 - does it make sense to separate PSM Interop server changes from the dualstack enablement env vars? I'd prefer that, but if not possible/desirable, I'm good with this too.